service/matching/tasklist/task_list_manager.go

```diff
@@ -491,10 +497,23 @@ func (c *taskListManagerImpl) AddTask(ctx context.Context, params AddTaskParams)
 	return syncMatch, err
 }
 
-// DispatchTask dispatches a task to a poller. When there are no pollers to pick
-// up the task or if rate limit is exceeded, this method will return error. Task
-// *will not* be persisted to db
+// DispatchTask dispatches a task to a poller on the active side. When there are no pollers to pick
+// up the task or if the rate limit is exceeded, this method will return error. Task
+// *will not* be persisted to db. On the passive side, dispatches the task to the taskCompleter; it will attempt
+// to complete the task if it has already been started.
 func (c *taskListManagerImpl) DispatchTask(ctx context.Context, task *InternalTask) error {
+	// optional configuration to enable cleanup of tasks, in the standby cluster, that have already been started
+	if c.config.EnableStandbyTaskCompletion() {
```

Comment:
Does this have the tasklist name embedded already? sorry I forget the keys for dynamic-config, but adding the tasklist name as a dimension we can do config rollout would be helpful

Reply:
oh, I didn't see below :+1: then